### ISSUE TITLE
[MSFT-125] Sample challenge page: Screen reader reads the registrants information in a textual format and not in a tabular format.

### ISSUE THREAD

**Issue by veshu** [importtest/importtest#125]
**Steps to Reproduce**
1. Go to the sample challenge page: https://clicktime.symantec.com/34CaebazimgmFxdkBZnihRc7Vc?u=https%3A%2F%2Fwww.topcoder.com%2Fchallenges%2F<sample-Sample challenge page:challengeid>?tab=details. 2. Ensure that the screen reader is switched on. 3. Navigate to the 'registrants' control using the tab key or the arrow key. 4. Press enter on the 'registrants' control. 6. Press the down arrow to access the registrants information. 7. The information is displayed in a tabular form, however screen reader doesn't recognise the tabular format.

**Expected Result**
Native tabular structure should be used so as the screen reader reads the data in a tabular format.

**Actual Result**
Screen reader reads the registrants information in a textual format and not in a tabular format.

**Device**: Desktop/Labtop
**Operating System**: Windows
**Browser**: Chrome

**WCAG Category**: 1. Perceivable
**WCAG Level**: A
**WCAG Success Criterion**: 1.3.1 Info and Relationships
**Screen Reader Used**: JAWS
**Accessibility Audit Tool used**: N/A

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=088f5f5d%2D9b1c%2D4b7e%2Dacbf%2D320c257c9a9a&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2FKameshwarid%2Fbug%2023
